rivery_cli/cli/rivers.py

```diff
@@ -400,7 +400,7 @@ def wait_for_end(session, run_id, timeout=3600 * 2):
 def check_run(ctx, **kwargs):
     """ Check the run status by runId """
     profile_name = ctx.get('PROFILE')
-    rivery_client = client.Client(name=profile_name)
+    rivery_client = client.Client(name=profile_name, profile=profile_name)
```

Comment:
Same as above:
This should come from the client from the context :)

Reply:
Talked F2F